src/components/shared/ResultList.css

```diff
@@ -113,10 +113,12 @@
   word-break: break-all;
   /** https://searchfox.org/mozilla-central/source/devtools/client/themes/variables.css **/
   color: var(--grey-40);
-  margin-left: 15px;
+  margin-left: 5px;
```

Comment:
Just curious what your motivation is for changing this margin? Is it just adding more space for the url? I'm not sure if we have a guideline for spacing in this content. I think the larger margin makes it a bit more readable, but I'd love to hear your thoughts on it 😄

Reply:
It felt like there is too much space when I added an ellipsis at the beginning, but now I am not so sure :D good point!
<img width="1067" alt="image" src="https://user-images.githubusercontent.com/8305170/54662029-c3d09c80-4adc-11e9-8d73-110cd06f80c7.png">

Reply:
I think we should keep left and right spacing consistent; IMO this change makes it too close to the right edge.

Reply:
Sure, I'll change it back.